### PR TITLE
Typos for 01_modelling.ipynb tutorial

### PR DESCRIPTION
**Pull request by georgebisbas**
Small typos while going through modelling tutorial

### CODE REVIEW

File: examples/seismic/tutorials/01_modelling.ipynb [hunk 1]
```diff
@@ -28,7 +28,7 @@
    "metadata": {},
    "source": [
     "## The acoustic seismic wave equation\n",
-    "The acoustic wave equation for the square slowness $m$ defined as $m=\\frac{1}{c^2}$, where $c$ is the speed of sound in the given physical media, and a source $q$ is given by:\n",
+    "The acoustic wave equation for the square slowness $m$, is defined as $m=\\frac{1}{c^2}$, where $c$ is the speed of sound in the given physical media, and a source $q$ is given by:\n",
```
Comment (FabioLuporini) [inline examples/seismic/tutorials/01_modelling.ipynb]: no comma before the verb

File: examples/seismic/tutorials/01_modelling.ipynb [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "\\end{cases}\n",
     "\n",
     "with the zero initial conditions to guarantee unicity of the solution.\n",
-    "The boundary conditions are Dirichlet conditions :\n",
+    "The boundary conditions are Dirichlet:\n",
```
Comment (FabioLuporini) [inline examples/seismic/tutorials/01_modelling.ipynb]: I preferred with repeated "conditions"/
Or: "The Dirichlet boundary conditions are:"

File: examples/seismic/tutorials/01_modelling.ipynb (hunk 1 above)
Comment (georgebisbas) [reply examples/seismic/tutorials/01_modelling.ipynb]: Fixed

File: examples/seismic/tutorials/01_modelling.ipynb (hunk 2 above)
Comment (georgebisbas) [reply examples/seismic/tutorials/01_modelling.ipynb]: Fixed

**Comment by FabioLuporini** [comment]
Thanks a lot, merged.

**Pull request merged**